pkg/bootstrapper/bootstrapper.go

```diff
@@ -114,6 +116,8 @@ type winNodeBootstrapper struct {
 	// ignitionFilePath is the path to the ignition file which is used to set up worker nodes
 	// https://github.com/coreos/ignition/blob/spec2x/doc/getting-started.md
 	ignitionFilePath string
```

Comment:
Do we need to maintain both options? Or will this be removed once we update the WMCO to use the newer option?

Reply:
We have to keep `--ignition-file-path` at least until we change WMCO to use the new one.
After that I think we could remove it safely.